### PR TITLE
Removed a broken link

### PR DESCRIPTION
**Pull request by pranav656**
A broken link referred to CIFAR-10 learning rate decay which was a broken link. So, I removed the link and the references to the link.

### CODE REVIEW

**Comment by vrv** [comment]
https://www.tensorflow.org/versions/r0.7/tutorials/deep_cnn/index.html

the image is there -- don't rely on github's .md files to look perfect :(
